packages/utils/src/testUtils.ts

```diff
@@ -120,8 +121,12 @@ function initSpec(Editor: typeof JWEditor, spec: TestEditorSpec, container: HTML
     document.body.appendChild(container);
     if (selection) {
         _setSelection(selection);
+    } else {
+        document.getSelection().removeAllRanges();
     }
-    return new Editor(container);
+    const editor = new Editor();
+    editor.configure(Dom, { target: container });
```

Comment:
There's a problem here. Nothing forces the user of this function to have the `Dom` plugin installed.